modules/dynamic-plugin-api.adoc

```diff
@@ -1690,6 +1589,135 @@ Deprecated: This hook is not related to console functionality. Hook that ensures
 
 :!power-bi-url:
 
+[discrete]
+== `VirtualizedTable`
+
+Deprecated: A component for making virtualized tables.
```

Comment:
Shall we point out the replacement together with deprecation note? 
reference [VirtualizedTable doc link](https://github.com/openshift/console/blob/main/frontend/packages/console-dynamic-plugin-sdk/docs/api.md#virtualizedtable)